### ISSUE TITLE
Add support for ExposureMode

### ISSUE THREAD

**Issue by ArkadiuszRaj** [AravisProject/aravis#487]
**Describe the solution you'd like**
We develop the firmware that need very precise start & stop of exposition. Defined by external GPS synced clock.
So the application is aware of the frame timestamp with requested precision.

We have a FLIR camera that supports ExposureMode setting with ability to steer start & stop of light acquisition.

**Describe alternatives you've considered**
SpinnakerAPI

**Additional context**
Add new enum ArvExposureMode with values like TIMED (using ExposureTime) and TRIGGER_WIDTH (using Frame Trigger signal pulse width).

Add new function:
```
void arv_camera_set_exposure_mode (ArvCamera *camera, ArvExposureMode value);
```

Excerpt from the camera xml file
```
    <Enumeration Name="ExposureMode" NameSpace="Standard">
        <ToolTip>Sets the operation mode of the Exposure.</ToolTip>
        <Description>Sets the operation mode of the Exposure.</Description>
        <DisplayName>Exposure Mode</DisplayName>
        <Visibility>Beginner</Visibility>
        <ImposedAccessMode>RW</ImposedAccessMode>
        <EnumEntry Name="Timed" NameSpace="Standard">
            <ToolTip>Exposure time is set using ExposureTime</ToolTip>
            <Description>Timed exposure. The exposure time is set using the ExposureTime or ExposureAuto features and the exposure starts with the FrameStart or LineStart.</Description>
            <DisplayName>Timed</DisplayName>
            <pIsImplemented>ExposureMode_Timed_Imp</pIsImplemented>
            <pIsAvailable>ExposureMode_Timed_Avl</pIsAvailable>
            <Value>1</Value>
        </EnumEntry>
        <EnumEntry Name="TriggerWidth" NameSpace="Standard">
            <ToolTip>Exposure time is width of Frame trigger signal pulse</ToolTip>
            <Description>Uses the width of the current Frame trigger signal pulse to control the exposure time.</Description>
            <DisplayName>Trigger Width</DisplayName>
            <pIsImplemented>ExposureMode_TriggerWidth_Imp</pIsImplemented>
            <pIsAvailable>ExposureMode_TriggerWidth_Avl</pIsAvailable>
            <Value>2</Value>
        </EnumEntry>
        <pValue>ExposureMode_Val</pValue>
    </Enumeration>
```

And arv-tool use:
```
$ arv-tool-0.8 control ExposureMode
FLIR-0138A959 (USB3)
ExposureMode = Timed

$ arv-tool-0.8 control ExposureMode=TriggerWidth
FLIR-0138A959 (USB3)
ExposureMode = TriggerWidth
```

**Comment by EmmanuelP** [comment]
Please try #488

**Comment by EmmanuelP** [comment]
Sould be fixed #488. Please reopen if there is an issue in the implementation.
